x-pack/plugins/observability/public/components/shared/page_template/README.md

```diff
@@ -17,6 +17,8 @@ Now within your solution's **public** plugin `setup` lifecycle method you can ca
 ```typescript
 // x-pack/plugins/example_plugin/public/plugin.ts
 
+import { of } from 'rxjs';
```

Comment:
@weltenwort WDYT about making this dep explicit? Not everyone is familiar with rxjs

Reply:
yes, that sounds like a good idea